src/EditorFeatures/Test2/FindReferences/FindReferencesTests.ParameterSymbol.vb

```diff
@@ -534,5 +534,107 @@ end class
 </Workspace>
             Await TestAPIAndFeature(input)
         End Function
+
+        <WorkItem(16757, "https://github.com/dotnet/roslyn/issues/16757")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        <Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.LocalFunctions)>
```

Comment:
I'd add some FAR tests too. 
The parameter tests are in "FindReferencesTests.ParameterSymbol", but perhaps local functions deserve their own file/partial